sync/src/main/java/tech/pegasys/artemis/sync/BlockPropagationManager.java

```diff
@@ -31,31 +33,37 @@
 import tech.pegasys.artemis.statetransition.blockimport.BlockImporter;
 import tech.pegasys.artemis.statetransition.events.block.ImportedBlockEvent;
 import tech.pegasys.artemis.storage.ChainStorageClient;
-import tech.pegasys.artemis.storage.events.SlotEvent;
 import tech.pegasys.artemis.util.async.SafeFuture;
 import tech.pegasys.artemis.util.collections.LimitedSet;
 import tech.pegasys.artemis.util.collections.LimitedSet.Mode;
+import tech.pegasys.artemis.util.time.SlotEvent;
+import tech.pegasys.artemis.util.time.SlotEventsChannel;
 
-public class BlockPropagationManager extends Service {
+public class BlockPropagationManager extends Service implements SlotEventsChannel {
   private static final Logger LOG = LogManager.getLogger();
 
   private final EventBus eventBus;
+  private final EventChannels eventChannels;
   private final ChainStorageClient storageClient;
   private final BlockImporter blockImporter;
-  private final PendingPool<SignedBeaconBlock> pendingBlocks;
+
+  @VisibleForTesting final PendingPool<SignedBeaconBlock> pendingBlocks;
```

Comment:
Why did this need to become visible for testing?

Reply:
Removed.

Reply:
Removed.